portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+public class Comment{
+    //personal data
+    private String firstName, lastName, email, phone;
+    //job related data: optional
+    private boolean jobRelated=false;
+    private String jobTitle="";
```

Comment:
Can we leave them as `null`, if they are optional? It would be more explicit than having just an empty string. We could then also annotate the optional fields with `@Nullable`. Another option would be to use `java.util.Optional<>`.

Reply:
I initialised jobTitle to null and added the @Nullable annotation